docs/adr/0008-non-eua-user-support.md

```diff
@@ -0,0 +1,37 @@
+# Support non-EUA users 
+
+[EASI - 2260](https://jiraent.cms.gov/browse/EASI-2260)
+
+Future use of the MINT application will extend to users who do not have an EUAID, specifically MAC (Medicare Administrative Contractor) users. Currently, the database uses the EUAID to distinguish a user for most tables.  Without an EUAID, a user is not currently able to modify data.
+
+The first iteration need of a MAC user is just authentication. This would still fit with our database paradigm, as there is no need to write information.
+
+In the future, MAC users might contribute to discussions, but there are no plans for allowing non-eua users to modify any other data, or collaborate on a model. Currently there is no distinction between an EUA user and a non EUA user. The IDM team advised that they could add a `is-eua: true` to the team if needed.
+
+
+*[decision drivers | forces]* <!-- optional -->
+Priority is given to the solution that gives us flexibility for anticipated future needs, as well as remaining as simple as possible.
+
+## Considered Alternatives
+
+* User Table
+* Conditional Secondary ID when needed
+
+## Decision Outcome
+
+* Chosen Alternative: User Table
+ After much discussion with the team, we have decided to implement a user table. This table is an investment in the future state of the application. Although not mandatory for the first implementation of MAC user support, it will allow greater flexibility for the experience provided to the entire user base.
+
+## Pros and Cons of the Alternatives <!-- optional -->
+
+### User Table
+
+* `+` Having a secondary user table allows us to store more additional data associated to a user.
```

Comment:
I think an additional `+` of the users table is that it can serve as a cache for user info that normally has to be fetched from an API (like CEDAR LDAP for getting users names & emails). Might be worth adding here!

Reply:
Agreed! Thanks Clay!